com.camunda.Modeler.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: com.camunda.Modeler
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: camunda-modeler-wrapper.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+
+add-extensions:
+  com.camunda.Modeler.Plugin:
+    directory: plugins
+    subdirectories: true
+    no-autodownload: true
+    autodelete: true
+
+modules:
+  - name: camunda-modeler
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/camunda-modeler
+      - cp -r camunda-modeler/* /app/share/camunda-modeler/
+      - install -Dm644 ${FLATPAK_ID}.desktop -t /app/share/applications
+      - install -Dm644 ${FLATPAK_ID}.appdata.xml -t /app/share/appdata
+      - install -Dm755 camunda-modeler-wrapper.sh /app/bin/camunda-modeler-wrapper.sh
+      - install -Dm644 camunda-modeler/support/icon_16.png /app/share/icons/hicolor/16x16/apps/${FLATPAK_ID}.png
+      - install -Dm644 camunda-modeler/support/icon_48.png /app/share/icons/hicolor/48x48/apps/${FLATPAK_ID}.png
+      - install -Dm644 camunda-modeler/support/icon_128.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - mkdir -p /app/plugins
+    sources:
+      - type: archive
+        only-arches: [x86_64]
+        url: https://github.com/camunda/camunda-modeler/releases/download/v5.32.0/camunda-modeler-5.32.0-linux-x64.tar.gz
+        sha256: 24e26b46fadfdcacfe39b0dfc986120157e8bd0d7965ca4d3cc1d727e82b66f0
+        strip-components: 1
+        dest: camunda-modeler
+      - type: file
+        path: com.camunda.Modeler.desktop
+      - type: file
+        path: com.camunda.Modeler.appdata.xml
+      - type: script
+        dest-filename: camunda-modeler-wrapper.sh
+        commands:
+          - export DISABLE_SANDBOX=true
```

Comment:
Instead of doing this we'd want to ensure that a respective app-armor profile is installed, cf. https://github.com/camunda/camunda-modeler/issues/4695#issuecomment-2478458250. Would that be possible?

Reply:
@nikku I have overlooked the proper solution of handling Electron apps in flatpaks. I have to use [zypak](https://github.com/refi64/zypak/blob/main/README.md) to execute the Electron app. This way flatpak can properly handle the sandboxing within the sandbox of flatpak itself (as discussed [here](https://github.com/flatpak/flatpak/issues/3044)).